ajax/libs/phaser/package.json

```diff
@@ -21,13 +21,13 @@
     "url": "https://github.com/photonstorm/phaser.git"
   },
   "license": "MIT",
-  "author": "Richard Davey <rdavey@gmail.com> (http://www.photonstorm.com)",
+  "author": "Richard Davey <rdavey@gmail.com> (https://www.photonstorm.com)",
```

Comment:
author's page doesn't support https.

Reply:
Hmmm. Strange.... From my side it is working...
![screen shot 2018-07-18 at 8 53 08](https://user-images.githubusercontent.com/11975561/42862447-31a114be-8a68-11e8-8fc2-88e567833bf9.png)

Reply:
@sashberd Your screenshot is the homepage?